### ISSUE TITLE
Proficiency check on slot validation

### ISSUE THREAD

**Issue by Mate2xo** [Mate2xo/THP-Creeons_la_coop#338]
### TLDR
Proficiency validation doesn't seem to be active on a Mission slot
We cannot update a member's slot if this slot is full

### As a 
User, or admin user

### When I'm
on the Mission creation/edit form

### Current behaviour
I can fill in the whole Mission slot with members that are not proficient on the cash register
I cannot update a member's time slot if the mission slot is full

### Desired behaviour
* The Mission slot should not be valid if there is not at least one assigned member that is proficient on the cash register
* We can update a member's time slot even if the mission slot is full
* Add a new proficiency level : `newbie` (`nouveau` in french)

### Additional infos
Maybe this only happens for admin users ? This validation was supposed to exist
